### PR TITLE
Landing page design and instructions added

### PR DESCRIPTION
**Pull request by Jiew23**
1. Instructions added on the landing page
2. I have added Bootstrap and make the page this way to make sure the webpage can be rendered on mobile phone with any width
3. This is now able to render correctly on both landscape and portrait
4. This version has been deployed to Heroku and also with the latest update in the refactor-app branch

### CODE REVIEW

**Comment by Jiew23** [comment]
@TinBryn I commented out at the time cuz the deployed landing page has everything overlapping on each other. I thought it was the viewport problem but after I deleted the viewport, nothing has changed. So I commented out the instruction div and deployed it to test if that is the only reason for the layout messing up.

File: static/styles/index.css [hunk 1]
```diff
@@ -2,69 +2,58 @@ body {
...
     background-repeat: no-repeat;
     background-attachment: fixed;
-    background-size: cover;
+    background-size: 100% 100%;
     padding: 0;
     margin: 0;
 }
 
-h1 {
+#header {
```
Comment (TinBryn) [inline static/styles/index.css:10]: This makes the title very large and on my phone it ends up with 1 or 2 words per line, and even then "application" doesn't fit

**Review by TinBryn — CHANGES_REQUESTED** [review]
The header is way too large on mobile it looks like this

> Welcome  
to the  
Molecule  
of the  
Month  
Web AR  
application

**Comment by Jiew23** [comment]
@TinBryn I feel this is the font size that suits the best on both the mobile app and PCs. But this is just my thinking. Do feel free to make any changes to the font size though.

**Pull request merged**